shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -649,6 +651,49 @@ public void systemInsetDisplayCutoutSimple() {
     assertEquals(100, viewportMetricsCaptor.getValue().viewInsetTop);
   }
 
+  @SuppressWarnings("deprecation")
+  @Test
+  @Config(sdk = 28)
```

Comment:
can this be minSdk?

Reply:
Yes, albeit with some extra modifications